.github/workflows/test-cov.yml

```
@@ -18,7 +18,7 @@ jobs:
 
       - uses: actions/setup-go@v5
         with:
-          go-version: "1.23.5"
+          go-version: "1.23.6"
```

can we use only the latest stable version here?

```suggestion
          go-version: "stable"
```

---

No, see https://github.com/ignite/cli/pull/4509